### ISSUE TITLE
AdNauseam filters are not updated

### ISSUE THREAD

**Issue by Vovadid** [dhowe/AdNauseam#1665]
AdNauseam filters are not updated at the click of a button. He writes that he is out of date, but 391 out of 409 remain and the update button is not removed next to it.
![Снимок](https://user-images.githubusercontent.com/39053957/82737352-3bc9d080-9d39-11ea-8cdc-5231aab1a882.PNG)
Windows 10 1909
Chrome Version 83.0.4103.61 (Official Build) (64-bit)
AdNauseam v3.9.104 uBlock Origin v1.20.0
Default Filter lists
Location: Russian

**Comment by cqx931** [comment]
Hi @Vovadid, sorry for getting back to you late. 
I'm not sure what you mean by "He writes that he is out of date", how do you make the conclusion that AdNauseam is still out of date after clicking the update button?

It's correct that the update button won't be removed after update, this will allow a forced update to be made just for AdNauseam filters without clearing the cache.

**Comment by Vovadid** [comment]
> Hi @Vovadid, sorry for getting back to you late.
> I'm not sure what you mean by "He writes that he is out of date", how do you make the conclusion that AdNauseam is still out of date after clicking the update button?
> 
> It's correct that the update button won't be removed after update, this will allow a forced update to be made just for AdNauseam filters without clearing the cache.

I shot a video for clarity. I will describe it again in words when I press the "update" button on this filter nothing happens.
Video: https://radikal.ru/video/5OtgeZ72hCt

**Comment by cqx931** [comment]
I just did a test on my end and the list itself does get updated correctly. 
If now, the first line of your adnauseam filters is "# Generics (any page)", then, the update button does work.

The numbers do seem weird though(409 used out of 411), I will look into it and see if there is anything I can fix there.

**Comment by dhowe** [comment]
Not related to this specific problem, but I've also felt the update page was somewhat confusing. Perhaps it would help to put hover text over the 3 buttons (most important perhaps would be  'purge all caches')

**Comment by cqx931** [comment]
1,2,4 are all designs we have inherited from ublock origin, so I also don't know the why behind these design decisions...
Changing the clock icon makes sense to me. 
I'm not sure what hover text can be added to the three buttons, any suggestions?

3. Looks should be a bug and this is how it looks like normally. I have opened another ticket for this issue.
![Screenshot 2020-06-16 at 1 04 58 PM](https://user-images.githubusercontent.com/3056154/84805267-fd3ed300-afd1-11ea-8ac6-42a4f3bb8d7b.png)
